### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by bertwesarg** [libunwind/libunwind#639] [General]
Dear @bregma,,I have the impression, that the development has settled for some time now. Maybe its time for the 1.8 release?

Thanks.

**Comment by bregma** [comment]
I was hoping to get #569 fixed before the 8.0 release, but I haven't been able to identify the cause (although I have some strong suspicions). Because it smells strongly of a race condition in the generic code I want to at least know the cause, although to be fair the issue has been there since at least 1.6 so it isn't really a regression.

Regardless of progress on #569 I intend to start the release process soon. Fixes can land on the branch too.
